lms/static/sass/multicourse/_about_pages.scss

```diff
@@ -44,11 +44,11 @@
       padding-bottom: 40px;
 
       .logo {
-        border-right: 1px solid rgb(200,200,200);
+        @include border-right( 1px solid rgb(200,200,200) );
```

Comment:
yeah off course, I did it only for ease of reading as there is a function within another but right now I start to remove these spaces.